### ISSUE TITLE
Requirements 

### ISSUE THREAD

**Issue by idanmoradarthas** [EthicalML/xai#3]
your requirements are very restrictive. Can you please change it to >= instead of ==.
for example:
```
numpy>=1.3
pandas>=0.23.0
matplotlib>2.02,<=3.0.3
scikit-learn>=0.19.0
```

**Comment by RafalSkolasinski** [comment]
I second this request!

**Comment by axsaucedo** [comment]
That's a good suggestion - thank you @idanmoradarthas and @RafalSkolasinski !

**Comment by axsaucedo** [comment]
Fixed via #13
